app/models/public_body.rb

```diff
@@ -216,6 +216,14 @@ def set_api_key!
     self.api_key = SecureRandom.base64(33)
   end
 
+  def self.find_by_name(name)
```

Comment:
it looks as though `PublicBody.find_by_name` is only used in specs so we could get rid of this relatively easily